app/src/components/Header.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Image, Text, View } from 'react-native';
 
 const Header = () => {
   return (
     <View className="items-center justify-between flex-row px-5 android:top-9">
-      <Text className="text-white font-medium font-zenMaru">Settings</Text>
+      <Image
+        source={require('../../assets/settings.png')}
```

Comment:
I think the right way should be importing the assset, so it does not dynamically require at each render